Tools/parametric_model/configs/fixedwing_longitudinal_model.yaml

```diff
@@ -0,0 +1,164 @@
+# general information
+model_name: "Gazebo Standard Plane"
+model_type: "Standard Plane Longitudinal Model"
+model_class:
+  "SimpleFixedWingModel"
+
+  # all vectors in FRD body frame if not specified otherwise
+model_config:
+  mass: 20.0
+  moment_of_inertia:
+    Ixx: 25.0
+    Iyy: 20.0
+    Izz: 20.0
+  actuators:
+    rotors:
+      # All rotors in the same group will share the coefficients
+      puller_:
+        - rotor_4:
+          description: "puller rotor"
+          rotor_type: "RotorModel"
+          dataframe_name: "u4"
+          rotor_axis:
+            - 1
+            - 0
+            - 0
+          turning_direction: -1
+          position:
+            - 0.22
+            - 0
+            - 0
+
+    control_surfaces:
+      wing_:
+        - control_surface_0:
+          description: "aileron_right"
+          dataframe_name: "u6"
+
+        - control_surface_1:
+          description: "aileron_left"
+          dataframe_name: "u5"
+
+        - control_surface_2:
+          description: "elevator"
+          dataframe_name: "u7"
+
+        - control_surface_3:
+          description: "rudder"
+          dataframe_name: "u2"
```

Comment:
Why does a longitudinal flight dynamics model have ailerons and rudders?

Reply:
Currently, they are just part of this model in order not to generate errors during the parsing. I'm however still planning on removing this dependency, so that they can be left out for this model at least.